dist/templates/ovnkube-master.yaml.j2

```diff
@@ -246,6 +246,8 @@ spec:
           value: "{{ ovn_hybrid_overlay_net_cidr }}"
         - name: OVN_DISABLE_SNAT_MULTIPLE_GWS
           value: "{{ ovn_disable_snat_multiple_gws }}"
+        - name: OVN_DISABLE_PKT_MTU_CHECK
+          value: "{{ ovn_disable_pkt_mtu_check }}"
```

Comment:
Not sure why `ovnkube-master` needs this environment variable since all the work is done by ovnkube-node

Reply:
done